decentraland/unity-renderer
# Fix: fixed visible selected parcel in minimap and player centering

## What does this PR change?

Fix #2795 
Disables the selected parcel icon in minimap and re-centers the player icon

...

## How to test the changes?


1. Go to: https://play.decentraland.zone/?renderer-branch=fix/minimap-offset
2. Verify the player position in map and minimap
3. Verify that in the minimap there isn't the selected parcel blue highlight

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
